src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/HttpRetryHandler.cs

```diff
@@ -2,17 +2,21 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Diagnostics;
 using System.Globalization;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
-using NuGet.Common;
+using NuGet.Logging;
 using NuGet.Protocol.Core.v3;
 
 namespace NuGet.Protocol
 {
     public class HttpRetryHandler
     {
+        public const string RequestLogFormat = "  {0} {1}";
+        public static readonly string ResponseLogFormat = "  {0} {1} {2}" + Strings.Milliseconds;
```

Comment:
I thought we decided during the original httpsource PR not to localize this. It's going to be confusing for an author to localize this. Unless this is going to be used on RTL I don't see a scenario where this information should move around.